nemo/src/io/format_builder.rs

```diff
@@ -489,6 +596,7 @@ pub(crate) enum AnyImportExportBuilder {
     Dsv(DsvBuilder),
     Rdf(RdfHandler),
     Json(JsonHandler),
+    Sparql(SparqlBuilder),
```

Comment:
This enum needs an `#[allow(variant_size_differences)]` (but GitHub won't let me annotate the enum itself, since it wasn't changed)